Code/Engine/Foundation/Utilities/CommandLineOptions.h

```diff
@@ -222,7 +222,7 @@ class EZ_FOUNDATION_DLL ezCommandLineOptionString : public ezCommandLineOptionDo
   ezCommandLineOptionString(const char* szSortingGroup, const char* szArgument, const char* szLongDesc, const char* szDefaultValue, bool bCaseSensitive = false);
 
   /// \brief Returns the value of this option. Either what was specified on the command line, or the default value.
-  ezString GetOptionValue(LogMode logMode, const ezCommandLineUtils* pUtils = ezCommandLineUtils::GetGlobalInstance()) const; // [tested]
+  const char* GetOptionValue(LogMode logMode, const ezCommandLineUtils* pUtils = ezCommandLineUtils::GetGlobalInstance()) const; // [tested]
```

Comment:
@jankrassnigg I changed the return value to const char* because you can easily end up with garbage strings if you return an ezString by value but then assign the result to a const char*.